solutions/advanced/cses-1695.mdx

```diff
@@ -85,4 +85,177 @@ int main() {
 
 </CPPSection>
 
+<JavaSection>
+```java
+import java.util.*;
+
+public class PoliceChase {
+	static final int N = 500;
+	static int n, m;
+	static int[] lev = new int[N];
+	static int[][] c = new int[N][N];
+	static int[] seen = new int[N];
+	static Queue<Integer> q = new LinkedList<>();
+
+	// BFS to find level of each node
```

Comment:
can you move these comments to the python code?

Reply:
Done